torch/ao/quantization/observer.py

```diff
@@ -1492,10 +1492,14 @@ def load_observer_state_dict(mod, obs_dict):
 
 # TODO(future PR): remove these defaults and enforce activation functions
 # to explicitly specify their output range
-default_symmetric_fixed_qparams_observer = FixedQParamsObserver.with_args(
+default_symmetric_fixed_qparams_range_neg1to1_observer = FixedQParamsObserver.with_args(
```

Comment:
should we remove `symmetric` from the name, since `neg1to1` has a more clear version of the same information?